playbook/app/pb_kits/playbook/pb_date_picker/_date_picker.tsx

```diff
@@ -78,6 +88,13 @@ const DatePicker = (props: DatePickerProps): React.ReactElement => {
     inputOnChange,
     inputValue,
     label = 'Date Picker',
+    margin,
+    marginBottom = "sm",
```

Comment:
@jasperfurniss I didn't really know how to make a default global prop so i set the value here

Reply:
reviewing this now. 👀

Reply:
@markdoeswork